app/models/auditing/JsonAuditModel.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.auditing
+
+import play.api.libs.json.JsValue
+
```

Comment:
Would you consider writing a test forJsonAuditModel.

Reply:
I'm not entirely sure what to test in honesty, the trait doesn't have any 'special' functions like parsing to/from JSON. It's simply defining the schema in which all audit models should adhere to.